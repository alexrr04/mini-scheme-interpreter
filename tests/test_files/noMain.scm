(define (sum x y)
    (+ x y)
)

(display (sum 10 5))
(newline)