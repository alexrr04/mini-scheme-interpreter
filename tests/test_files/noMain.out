Error: No main function defined in file tests/test_files/noMain.scm